test/system/entities_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'application_system_test_case'
+
+class EntitiesTest < ApplicationSystemTestCase
+  setup do
+    @entity = entities(:one)
+  end
+
+  test 'visiting the index' do
+    visit entities_url
```

Comment:
- [ ] Kindly fix your failing test. I can see that before your first review, all your tests were passing but after making the required changes we have a failing test, please do so.

Before
![image](https://github.com/jkanyi-web/BudgetBuddy/assets/81932478/e3048741-f1f2-441f-b95c-8082f37deb7b)

After
![image](https://github.com/jkanyi-web/BudgetBuddy/assets/81932478/087d9a58-3e70-4d20-8856-b358dbd5d4e3)